src/Dialect/ONNX/ONNXOpsHelper.cpp

```diff
@@ -0,0 +1,42 @@
+//====------ ONNXOpsHelper.cpp - Helper functions for ONNX dialects -------===//
+//
+// Copyright 2019 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file contains helper functions for lowering ONNX ops to Krnl Dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Dialect/ONNX/ONNXOpsHelper.hpp"
```

Comment:
Simply use `#include "ONNXOpsHelper.hpp"`? I think it has been a convention in our coding style to not include the full path to the header file that contains the declaration of the functions currently being implemented.